src/core/reactor_backend.cc

```diff
@@ -1140,6 +1140,8 @@ try_create_uring(unsigned queue_len, bool throw_on_error) {
             IORING_OP_FSYNC,
             IORING_OP_SENDMSG,  // linux 5.3
             IORING_OP_RECVMSG,
+            IORING_OP_ACCEPT,
+            IORING_OP_CONNECT,
             IORING_OP_READ,     // linux 5.6
             IORING_OP_WRITE,
             IORING_OP_SEND,
```

Comment:
I think you need to check for IORING_FEAT_SUBMIT_STABLE too. Otherwise connect() can forget the address if it gets blocked. It was in 5.5 so we can require it.